### PR TITLE
JAMES-2855 Upgrade Junit

### PR DESCRIPTION
**Pull request by trantienduchn**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -607,10 +607,10 @@
         <commons-logging.version>1.1.1</commons-logging.version>
 
         <dnsjava.version>2.1.8</dnsjava.version>
-        <junit.version>4.12</junit.version>
-        <junit.jupiter.version>5.2.0</junit.jupiter.version>
-        <junit.plateform.version>1.4.1</junit.plateform.version>
-        <junit.vintage.version>5.2.0</junit.vintage.version>
+        <junit.version>4.13-beta-3</junit.version>
```
Comment (chibenwa) [inline pom.xml]: Not sure "beta" is a good idea...

Comment (trantienduchn) [reply pom.xml]: so, the last final version is still 4.12. I will drop that commit

**Comment by chibenwa** [comment]
:green_apple: before CI reboot

**Comment by trantienduchn** [comment]
The junit 5 version is almost one year late, I guess there are many changes but IDK which, so I want to merge this one first and then make another PR for refactoring the code to get it match with the newest features from junit 5.

**Comment by chibenwa** [comment]
test this please

**Pull request closed without merging**